src/templates/docs/font-weight/index.html

```diff
@@ -50,20 +50,20 @@ <h2 class="f3 bold">Examples</h2>
 
         <h3 class="f5 book sans-serif pt4 caps black-100">String Values</h3>
         <div class="cf">
-          <div class="fl f5 f2-m f1-l normal v-mid tc"><span class="db bg-near-white pv3 ba b--white ph4">Aa <span class="db f6 fw4">normal</span></span></div>
-          <div class="fl f5 f2-m f1-l b v-mid tc"><span class="db bg-near-white pv3 ba b--white ph4">Aa <span class="db f6 fw4">b</span></span></div>
+          <div class="fl f5 f2-m f1-l normal v-mid tc"><span class="db bg-near-white pv3 ba b--white ph4">Aa <code class="f6 db">.normal</code></span></div>
```

Comment:
Adding class to all `font-weight` examples, but left in numeric values as I think it's still useful for the user to understand what it correlates to